blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -0,0 +1,159 @@
+
+	if (!elements.length) return null;
+
+	// const getParentClassNames = () => {
+	// 	if (elements && elements.length > 4) {
+	// 		return `${BLOCK_CLASS_NAME} ${BLOCK_CLASS_NAME}__story-card-5-plus`;
+	// 	}
+	// 	return `${BLOCK_CLASS_NAME} ${BLOCK_CLASS_NAME}__story-card-4-less`;
+	// };
+
+	return (
+		// <Stack className={getParentClassNames()}>
+		<Stack className={BLOCK_CLASS_NAME}>
+			{headline && headline.trim() !== "" && <Heading>{headline}</Heading>}
+
+			{elements && elements.length && (
+				<Carousel
+					id={identifier}
+					label={carouselLabel}
+					slidesToShow={slidesToShow}
```

Comment:
Remove this prop and the component will work as you expect it too - This prop will set the slidesToShow for all viewports

Reply:
Perfect!